website/docs/getting-started/cluster-setup/kubernetes.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Kubernetes
+sidebar_position: 1
+---
+
+# Setup a Kubernetes cluster
+*Note that this guide is only helpful in setting up a development environment; this setup is not recommended for a production environment.*
+
+## Prerequisites
+* This guide assumes that you have a Kubernetes cluster setup, this could also be a [minikube](https://minikube.sigs.k8s.io/docs/start/) cluster.
+* This guide also assumes that you have admin privileges to the cluster, since the operator installation is only possible with an admin user.
+
+## Enable Ingress
+To access an application externally, odo creates a URL; installing an ingress-controller helps in using this feature on a Kubernetes cluster.
+
+To enable the Ingress feature on a Kubernetes cluster _other than minikube_, run the following command to install the default setup:
+```shell
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v0.48.1/deploy/static/provider/cloud/deploy.yaml
```

Comment:
Please indicate that this command is specific to the `nginx` ingress controller. Also, you could point to the upstream doc for more details on installation: https://kubernetes.github.io/ingress-nginx/deploy/